odoo/addons/base/models/ir_cron.py

```diff
@@ -515,3 +515,7 @@ class ir_cron_trigger(models.Model):
 
     cron_id = fields.Many2one("ir.cron", index=True)
     call_at = fields.Datetime()
+
+    @api.autovacuum
+    def _gc_cron_triggers(self):
+        self.search([('call_at', '<', datetime.now() + relativedelta(weeks=-1))]).unlink()
```

Comment:
Weird, all old triggers should be removed by `process_job`

https://github.com/odoo/odoo/blob/9e8b2ba1ae05de72f492d833edd8da91ded6f5b0/odoo/addons/base/models/ir_cron.py#L343-L347

In case there are cron triggers that keep accumulating on odoo.com then my gut feeling tells me that there is something wrong.

Still I'm ok we merge this autovacuum if it's the simplest way to solve the problem :)

Reply:
#113575